src/Service/OpenamApiClient.php

```diff
@@ -16,6 +16,16 @@
  */
 class OpenamApiClient {
 
+  const HTTP_STATUS_CODE_MESSAGES = array(
```

Comment:
perhaps update using standards `array()`

Reply:
Yes this I wasn't sure about and hadn't used before as CONST Arrays were introduced in PHP7.

The only reason I used a CONST instead was so they could not be changed, as they should be fixed. These are defined by OpenAM.

https://backstage.forgerock.com/docs/am/5.5/dev-guide/#about-crest-response-codes